storage/interceptors/smaap_service_instance_interceptor.go

```diff
@@ -514,6 +533,11 @@ func (i *ServiceInstanceInterceptor) pollServiceInstance(ctx context.Context, os
 		key = &opKey
 	}
 
+	// epsilontal todo: review if in the right place:
+	if instance.ReferencedInstanceID != "" {
```

Comment:
polling is not applicable for refference instances, this line should be probably removed.
do we have a test that covers this case?

Reply:
I'll keep it open, we need to consult here.

Reply:
remove it you won't poll here because you don't involve broker